src/Tools/BuildBoss/ProjectCheckerUtil.cs

```diff
@@ -326,6 +326,7 @@ private bool CheckTargetFrameworks(TextWriter textWriter)
                     case "netcoreapp1.1":
                     case "netcoreapp2.1":
                     case "netcoreapp3.0":
```

Comment:
Do we still use these in the build? Consider removing any that we are no longer using.

Reply:
I think there are a few I can remove here.